docs/unified-test-documentation/dasharo-security/20P-boot-menu.md

```diff
@@ -0,0 +1,97 @@
+# Dasharo Security: Boot menu enable/disable
```

Comment:
Is the test suite nomenclature corresponds to the accepted product ID nomenclature?

Reply:
As for the Product-id list, I didn't find such a module there, so I named it as analogous to other modules as possible.